sdk/servicebus/service-bus/src/old/oldServiceBusReceiverClient.ts

```diff
@@ -0,0 +1,358 @@
+import Long from "long";
+import { InternalReceiver, InternalSessionReceiver } from "../internalReceivers";
+import {
+  ServiceBusClientOptions,
+  createConnectionContextForConnectionString,
+  createConnectionContextForTokenCredential
+} from "../old/serviceBusClient";
+import {
+  TokenCredential,
+  OnMessage,
+  OnError,
+  MessageHandlerOptions,
+  CorrelationFilter,
+  RuleDescription
+} from "..";
+import { isTokenCredential } from "@azure/core-amqp";
+import { ClientEntityContext } from "../clientEntityContext";
+import { ClientType } from "../client";
+import { ReceiveMode, ServiceBusMessage, ReceivedMessageInfo } from "../serviceBusMessage";
+import { SessionReceiverOptions } from "../session/messageSession";
+import { generate_uuid } from "rhea-promise";
+import { ConnectionContext } from "../connectionContext";
+
+export type ServiceBusClientReceiverOptions = ServiceBusClientOptions & SessionReceiverOptions;
+
+/**
+ * @internal
+ * @ignore
+ */
+export class ServiceBusReceiverClient {
+  public _receiveMode: ReceiveMode;
```

Comment:
is it common to have underscores before public properties? or even private properties? Seems a bit unnecessary.

Reply:
No, you're 100% right. That's not even supposed to be public.

Reply:
Hm..I just realized that's in the _old_ service bus client. @HarshaNalluru  is working on removing that client altogether.